server/tinylicious/src/routes/ordering/documents.ts

```diff
@@ -59,7 +59,8 @@ export function create(storage: IDocumentStorage): Router {
 			`http://${request.hostname}`,
 			`http://${request.hostname}`,
 			values,
-			false,
+			false /* enableDiscovery */,
+			false /* isEphemeralContainer */,
```

Comment:
@NicholasCouri Is this enough or do we need to implement something else for ephemeral containers?

Reply:
It should be enough - nothing unless we implement it for t9s, which is not the case.